rest/admin-v2/rest/src/main/java/org/keycloak/admin/api/client/DefaultClientApi.java

```diff
@@ -105,12 +108,12 @@ public ClientRepresentation patchClient(JsonNode patch, FieldValidation fieldVal
         }
     }
 
-    static void validateUnknownFields(FieldValidation fieldValidation, ClientRepresentation rep, HttpResponse response) {
+    static void validateUnknownFields(ClientRepresentation rep, HttpResponse response) {
         if (!rep.getAdditionalFields().isEmpty()) {
-            if (fieldValidation == null || fieldValidation == FieldValidation.Strict) {
+            if (DEFAULT_FIELD_VALIDATION == null || DEFAULT_FIELD_VALIDATION == FieldValidation.Strict) {
```

Comment:
I'd rather prefer we removed also the `FieldValidation` enum. We do not know if we'll even make it configurable (that depends on the result of #44161). We do not need it for now (just like the SPIs we removed in the previous PR).

Reply:
Ok, we'll add it back later.